include/openenclave/enclave.h

```diff
@@ -22,6 +22,7 @@
 #include "bits/report.h"
 #include "bits/result.h"
 #include "bits/types.h"
+#include "corelibc/bits/defs.h"
```

Comment:
Looking at the installed headers, this one is actually not installed. Can you move OE_NO_RETURN to `include/bits/defs.h` and then just include that header here instead?

Reply:
Done with it.